### ISSUE TITLE
Implement a lightweight survey banner

### ISSUE THREAD

**Issue by alexgibson** [mozilla/kitsune#3490]
The current survey on the SUMO homepage looks like a pretty lightweight thing to show users:

![screenshot_2019-03-08 mozilla support](https://user-images.githubusercontent.com/400117/54044983-434e9980-41c8-11e9-861b-1f6eec679e09.png)

Taking a look under the hood, this banner weighs in over 425Kb, makes 7 requests, and seems to include over 300Kb of JavaScript as well as a custom google-hosted font.

On slower connections, this has the effect of increasing page load time by around 4 seconds

Before: https://www.webpagetest.org/results.php?test=190306_WY_68869404a3c3fad15e459e9c5a643afb

After: https://www.webpagetest.org/results.php?test=190308_P6_b5a4f24875211063ae27603b7d35fe3d

A custom script that loads a survey gizmo banner/link could likely be made that weighs a fraction of the cost.

**Comment by pmac** [comment]
I'll add that it's very annoying UX-wise since it doesn't go away when one hits the <ESC> key.

**Comment by akatsoulas** [comment]
The banner is no longer a popup and a there is some research to replace this altogether. This effort will be tracked in a separate issue.
